data/splitContent.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "type": "splitContent",
+    "props": {
+      "children": [
+        [
```

Comment:
OK, maybe we pass a `variant` to signal to front-end what to do with this component.